esda/__init__.py

```diff
@@ -3,6 +3,10 @@
 =================================================
 
 """
+
+import contextlib
+from importlib.metadata import PackageNotFoundError, version
+
 from . import _version, adbscan, shape  # noqa F401
```

Comment:
```suggestion
from . import adbscan, shape  # noqa F401
```

this probably should not be there?

Reply:
@knaaptime Did you see Martin's comment here?